modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/shotgun.dm

```diff
@@ -0,0 +1,63 @@
+// SolFed shotgun (this was gonna be in a proprietary shotgun shell type outside of 12ga at some point, wild right?)
+
+/obj/item/gun/ballistic/shotgun/riot/sol
+	name = "\improper Carwo 'Renoster' Shotgun"
+	desc = "A twelve gauge shotgun with an eight shell capacity underneath. Made for and used by SolFed's various military branches."
+
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/company_and_or_faction_based/carwo_defense_systems/guns48x.dmi'
+	icon_state = "renoster"
+
+	worn_icon = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/carwo_defense_systems/guns_worn.dmi'
+	worn_icon_state = "renoster"
+
+	lefthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/carwo_defense_systems/guns_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/carwo_defense_systems/guns_righthand.dmi'
+	inhand_icon_state = "renoster"
+
+	inhand_x_dimension = 32
+	inhand_y_dimension = 32
+
+	fire_sound = 'modular_skyrat/modules/modular_weapons/sounds/shotgun_heavy.ogg'
+	suppressed_sound = 'modular_skyrat/modules/modular_weapons/sounds/suppressed_heavy.ogg'
+	can_suppress = TRUE
+
+	suppressor_x_offset = 11
+
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK | ITEM_SLOT_OCLOTHING
+
+/obj/item/gun/ballistic/shotgun/riot/sol/Initialize(mapload)
+	. = ..()
+
+	SET_BASE_PIXEL(-8, 0)
```

Comment:
Yeah it probably should be a define, if it's going to be repeated everywhere like that.